---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

You can also find my articles on my [Google Scholar Profile](https://scholar.google.com/citations?user=9517icQAAAAJ&hl=en).

{% include base_path %}
## Pre-prints and Manuscripts Under Review
- [2023] Sejour R, Leatherwood J, **Yurovsky A**, Futcher B. (2023) No Ramp Needed: Spandrels, Statistics, and a Slippery Slope. bioRxiv. 2023 Aug 19. doi: 10.1101/2022.06.27.497802.

## Peer-Reviewed Journal Articles and Conference Proceedings
- [2024] Sejour R, Leatherwood J, **Yurovsky A**, Futcher B. (2024) Enrichment of rare codons at 5' ends of genes is a spandrel caused by evolutionary sequence turnover and does not improve translation. eLife. 2024 Jul 15. 12:RP89656. doi: 10.7554/eLife.89656.3
- [2023] Hegelmeyer NK, Parkin LA, Previti ML, Andrade J, Utama R, Sejour RJ, Gardin J, Muller S, Ketchum S, **Yurovsky A**, Futcher B, Goodwin S, Ueberheide B, Seeliger JC. Gene recoding by synonymous mutations creates promiscuous intragenic transcription initiation in mycobacteria. mBio. 2023 Oct 31;14(5):e0084123. doi: 10.1128/mbio.00841-23. PubMed PMID: 37787543.
- [2023] Sweeney MD, Torre-Healy LA, Ma VL, Hall MA, Chrastecka L, **Yurovsky A**, Moffitt RA. (2023) FaStaNMF: a Fast and Stable Non-negative Matrix Factorization for Gene Expression. IEEE/ACM Trans Comput Biol Bioinform. Epub ahead of print. 2023/07/19; doi: 10.1109/TCBB.2023.3296979. PubMed PMID: 37467096.
- [2022] **Yurovsky A**, Gardin J, Futcher B, Skiena S. (2022) Statistical Methodology for Ribosomal Frameshift Detection. Proceedings of the 2022 ACM International Conference on Bioinformatics, Computational Biology and Health Informatics (ACM-BCB). 12:1-10. doi: 10.1145/3535508.3545529.
- [2019] **Yurovsky A**, Futcher B, Skiena S. (2019) Pre-Phaser: Precise Cell-Cycle Phase Detector for scRNA-seq. Proceedings of the 2019 ACM International Conference on Bioinformatics, Computational Biology and Health Informatics (ACM-BCB). 376-382 doi: 10.1145/3307339.3342174.
- [2018] **Yurovsky A**, Amin MR, Gardin J, Chen Y, Skiena S, Futcher B. (2018) Prokaryotic coding regions have little if any specific depletion of Shine-Dalgarno motifs. PLoS One. 13(8):e0202768. doi: 10.1371/journal.pone.0202768. PubMed PMID: 30138485.
- [2018] Amin MR, **Yurovsky A**, Chen Y, Skiena S, Futcher B. (2018) Re-annotation of 12,495 prokaryotic 16S rRNA 3â€™ ends and analysis of Shine-Dalgarno and anti-Shine-Dalgarno sequences. PLoS One 13(8):e0202767. doi: 10.1371/journal.pone.0202767. PubMed PMID: 30138483.
- [2018] Amin MR, **Yurovsky A**, Tian Y, Skiena S. (2018) DeepAnnotator: Genome Annotation with Deep Learning. Proceedings of the 2018 ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics (ACM-BCB). 2018;254-259. doi: 10.1145/3233547.3233577.
- [2015] Waszak SM, Delaneau O, Gschwind AR, Kilpinen H, Raghav SK, Witwicki, RM, Orioli A, Wiederkehr M, Panousis NI, **Yurovsky A**, Romano-Palumbo L, Planchon A, Bielser D, Padioleau I, Udin G, Thurnheer S, Hacker D, Hernandez N, Reymond A, Deplancke B, Dermitzakis ET. (2015) Population variation and genetic control of modular chromatin architecture in humans. Cell. 2015;162(5):1039-1050. doi: 10.1016/j.cell.2015.08.001. PubMed PMID: 26300124.
- [2015] Futcher B, Gorbatsevych O, Shen SH, Stauft CB, Song Y, Wang B, Leatherwood J, Gardin J, **Yurovsky A**, Mueller S, Wimmer E. (2015) Reply to Simmonds et al.: Codon pair and dinucleotide bias have not been functionally distinguished. Proceedings of the National Academy of Sciences. 2015/07/14;112(28):E3635-6. doi: 10.1073/pnas.1507710112. PubMed PMID: 26071446.
- [2015] Shen SH, Stauft CB, Gorbatsevych O, Song Y, Ward CB, **Yurovsky A**, Mueller S, Futcher B, Wimmer E. (2015) Large-scale recoding of an arbovirus genome to rebalance its insect versus mammalian preference. Proceedings of the National Academy of Sciences. 112(15):4749-4754. doi: 10.1073/pnas.1502864112. PubMed PMID: 25825721.
- [2015] Gutierrez-Arcelus M, Ongen H, Lappalainen T, Montgomery SB, Buil A, **Yurovsky A**, Bryois J, Padioleau I, Romano L, Planchon A, Falconnet E, Bielser D, Gagnebin M, Giger T, Borel C, Letourneau A, Makrythanasis P, Guipponi M, Gehrig C, Antonarakis SE, Dermitzakis ET. Tissue-specific effects of genetic and epigenetic variation on gene regulation and splicing. PLoS Genet. 2015 Jan 29;11(1):e1004958. doi: 10.1371/journal.pgen.1004958. PMID: 25634236.
- [2014] - Gardin J, Yeasmin R, **Yurovsky A**, Cai Y, Skiena S, Futcher B.  (2014) Measurement of average decoding rates of the 61 sense codons in vivo. eLife. 2014 Oct 27:3:e03735. doi: 10.7554/eLife.03735. PubMed PMID: 25347064.
- [2014] - Waszak SM, Kilpinen H, Gschwind AR, Orioli A, Raghav SK, Witwicki RM, Migliavacca E, **Yurovsky A**, Lappalainen T, Hernandez N, Reymond A, Dermitzakis ET, Deplancke B. (2014) Identification and removal of low-complexity sites in allele-specific analysis of ChIP-seq data. Bioinformatics. 2014 Jan 15;30(2):165-71. doi: 10.1093/bioinformatics/btt667. PubMed PMID: 24255646.
- [2013] - Kilpinen H, Waszak SM, Gschwind AR, Raghav SK, Witwicki RM, Orioli A, Migliavacca E, Wiederkehr M, Gutierrez-Arcelus M, Panousis NI, **Yurovsky A**, Lappalainen T, Romano-Palumbo L, Planchon A, Bielser D, Bryois J, Padioleau I, Udin G, Thurnheer S, Hacker D, Core LJ, Lis JT, Hernandez N, Reymond A, Deplancke B, Dermitzakis ET. (2013) Coordinated effects of sequence variation on DNA binding, chromatin structure, and transcription. Science. 2013;342(6159):744-7. doi: 10.1126/science.1242463. PubMed PMID: 24136355.
- [2013] - Gutierrez-Arcelus M, Lappalainen T, Montgomery SB, Buil A, Ongen H, **Yurovsky A**, Bryois J, Giger T, Romano L, Planchon A, Falconnet E, Bielser D, Gagnebin M, Padioleau I, Borel C, Letourneau A, Makrythanasis P, Guipponi M, Gehrig C, Antonarakis SE, Dermitzakis ET. (2013) Passive and active DNA methylation and the interplay with genetic variation in gene regulation. Elife. 2013 Jun 4:2:e00523. doi: 10.7554/eLife.00523. PubMed PMID: 23755361.
- [2011] - **Yurovsky A**, Moret BME. (2011) FluReF, an automated flu virus reassortment finder based on phylogenetic trees. BMC Genomics. 2011;12 Suppl 2(Suppl 2):S3. doi: 10.1186/1471-2164-12-S2-S3. PubMed PMID: 21989112.
- [2010] - **Yurovsky A**, Moret BME. FluRF, an automated flu virus reassortment finder based on phylogenetic trees. 2010 IEEE International Conference on Bioinformatics & Biomedicine (BIBM-2010)
- [2005] - **Yurovsky A**, Gopalakrishnan P. A new Cell-Planning based automatic placement technique for optimization of Analog, RF, and mixed-signal circuit layouts. Cadence Technical Conference, May 2005.


### US Patents

- [2019] Futcher B, Gardin J, Skiena S, **Yurovsky A**, Wimmer E, Mueller S. Modified protein encoding sequences having increased rare hexamer content. US Patent App. 15/773,685.
- [2011] Colwell R, Lamant GSC, **Yurovsky A**, Rosek T. Registry for Electronic Design Automation of Integrated Circuits. Assignee: Cadence Design Systems. Issued on 05/17/2011.  
- [2010] Gopalakrishnan P, **Yurovsky A**. Optimizing Circuit Layouts by Configuring Rooms for Placing Devices Assignee: Cadence Design Systems. Issued on 02/16/2010.

